cnxepub/collation.py

```diff
@@ -33,7 +33,8 @@ def easybake(ruleset, in_html, out_html):
     with respective read and write ability.
 
     """
-    html = etree.parse(in_html)
+    html_parser = etree.HTMLParser(encoding="utf-8")
```

Comment:
Umm, @philschatz , I respect your coding chops, but I doubt it's a quick fix. The extra gnarly bit is that the `sort-by/group-by` code isn't even using `cssselect2` right now, but `cssselect`'s `xpath` functionality. That needs to be converted over as well.